pages/resources/guild.mdx

```diff
@@ -1357,6 +1357,51 @@ Defines the criteria used to satisfy Onboarding constraints that are required fo
 }
 ```
 
+### Premium Subscription Object
+
+A premium subscription represents a member's premium subscription ([boost](https://support.discord.com/hc/en-us/sections/360007875211-Server-Boosting)) in a guild.
+
+###### Premium Subscription Structure
+
+| Field             | Type                                                              | Description                                                         |
+| ----------------- | ----------------------------------------------------------------- | ------------------------------------------------------------------- |
+| id                | snowflake                                                         | The ID of the subscription                                          |
```

Comment:
```suggestion
| id                | snowflake                                                         | The ID of the premium guild subscription                                          |
```
be specific here cuz there's a lot of IDs this could be referring to (e.g. the overall user sub, the user's premium guild sub slot, or the guild sub itself)